### ISSUE TITLE
Very short dropouts in general LED output

### ISSUE THREAD

**Issue by fightforlife** [hyperion-project/hyperion.kodi#52]
Hey there,

I am using this plugin for while and really like it since grabbing efficiency seems to be better than xcb or x11. (Lower CPU load)
But there is a small problem that is bugging me and I do not know how to debug this:
Every few seconds (3-10, not always the same) all LEDs at once get a little bit darker, as if a signal was missing for 200ms.

I already tried hyperion.ng with Adalight on an Arduino Nano and hyperhdr with hyperserialesp8266. The visual performance seem to be very similar with normal smoothing enabled. But the short dropouts exist on both, so I guess this could be a grabber Problem.
If I disable smoothing then the LEDs go dark for 200-500ms.

What is the best way to debug this from the grabber side?

**Comment by fightforlife** [comment]
Other interesting points:
- the frame rate settings does not change the grabbing speed. Setting it to 1 still works fine 
- it seems the LEDs are a very small bit faster than the Kodi player if not smoothing is enabled.

**Comment by chriss158** [comment]
I debugged the issue. Looks like it's the same issue i've described in this post.
https://forum.kodi.tv/showthread.php?tid=338046&pid=2799097#pid2799097

At that time I was able to identify the faulty captures. But with the current kodi/hyperion plugin version I can't see any differences to the normal captures. At that time I could ignore the faulty captures (see here: https://github.com/chriss158/hyperion.kodi/tree/flickering-workaround).

There is also a very old kodi issue:
https://github.com/xbmc/xbmc/issues/15153
